tests/components/MCQ.test.ts

```diff
@@ -17,9 +18,31 @@ describe("MCQ.vue", () => {
     const wrapper = mount(MCQ, {
       props: {
         title,
+        options,
       },
     });
     // Rather than looking at the whole wrapper, targetting specific class.
     expect(wrapper.get(".mcq-title").text()).toContain(title);
   });
+  test("Renders component with options", () => {
+    const wrapper = mount(MCQ, {
+      props: {
+        title,
+        options,
+      },
+    });
+
+    const questionKeys = Object.keys(options);
+
+    // Check that the correct number of options are rendered
+    const optionList = wrapper.findAll(".mcq-option");
+    expect(optionList.length).toBe(questionKeys.length);
+
+    // Check that each option is rendered correctly
+    for (const [index] of questionKeys.entries()) {
+      const renderedOption = optionList[index];
+      const [key, value] = Object.entries(options)[index];
+      expect(renderedOption.text()).toBe(`${key}: ${value}`);
+    }
+  });
```

Comment:
Test impacted by changing `options` struct. 
Also test for 0,1 options passed.
Consider if optionView becomes a seperate component - delay until there is percevied benefits.